nesc/whip6/platforms/boards/cc26xxbased/cherry-v1/scif_uart/sce_uart_emulator.lst

```diff
@@ -0,0 +1,948 @@
+               .deffile adi_defs.asm

```

Comment:
I thought it would be nice to have the source code for sensor controller at the same place. Can delete if that's not necessary.